modules/swagger-codegen/src/main/resources/python/__init__model.mustache

```diff
@@ -1,9 +1,10 @@
 # coding: utf-8
 
+# flake8: noqa
 {{>partial_header}}
 
 from __future__ import absolute_import
 
 # import models into model package
-{{#models}}{{#model}}from .{{classFilename}} import {{classname}}{{/model}}
+{{#models}}{{#model}}from {{modelPackage}}.{{classFilename}} import {{classname}}{{/model}}
```

Comment:
@wing328 this change broke our use case - we generate code and put it inside our library, so we don't really install swagger generated code itself. so now we get import errors because "swagger_client" is missing

Reply:
`swagger_client` is just the default top level package name, as part of the inputs when generating the code you can specify the package name to match the package name you leverage within your code.

Reply:
current structure:
some_client/swagger_client
some_client/__init__.py
some_client/client.py - where we import swagger_client stuff
....

change to the name will not fix the problem unfortunately, we need those relative imports :)

Reply:
I just opened #8738 to address the need for relative imports